src/styles/style.scss

```diff
@@ -6,3 +6,20 @@
   @extend .fa-3x;
 }
 
+.card {
+  padding:1.5rem;
+  box-shadow:0 1px 2px #aaa;
+  background:white;
+  margin:0 1rem 1rem;
+  border-radius:3px;
+}
+
+.container{
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.card-size {
```

Comment:
the container stuff can't as it needs all the cards to be in one container rather than all having separate ones